server/form-pages/apply/about-the-person/personal-information/gender.test.ts

```diff
@@ -24,7 +24,8 @@ describe('Gender', () => {
       const page = new Gender({ gender: null }, application)
 
       expect(page.errors()).toEqual({
-        gender: `Choose either Yes, No or Prefer not to say`,
+        gender:
+          "Select if the gender they identify with is the same as the sex registered at birth or 'Prefers not to say'",
```

Comment:
The option is "Prefer" instead of "Prefers". Not sure which one is more grammatically correct, but they should match.